parachain/src/chain_spec/local.rs

```diff
@@ -26,84 +23,81 @@ use artemis_core::AssetId;
-	let mut props = Properties::new();
-	props.insert("tokenSymbol".into(), "DEV".into());
-	props.insert("tokenDecimals".into(), 12.into());
+    let mut props = Properties::new();
+    props.insert("tokenSymbol".into(), "DEV".into());
+    props.insert("tokenDecimals".into(), 12.into());
 
-	ChainSpec::from_genesis(
-		"Artemis Local Testnet",
-		"local_testnet",
-		ChainType::Local,
-		move || {
-			testnet_genesis(
-				vec![
-					get_account_id_from_seed::<sr25519::Public>("Alice"),
-					get_account_id_from_seed::<sr25519::Public>("Bob"),
-					get_account_id_from_seed::<sr25519::Public>("Charlie"),
-					get_account_id_from_seed::<sr25519::Public>("Dave"),
-					get_account_id_from_seed::<sr25519::Public>("Eve"),
-					get_account_id_from_seed::<sr25519::Public>("Ferdie"),
-					get_account_id_from_seed::<sr25519::Public>("Relay"),
-					get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
-					get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
-				],
-				para_id
-			)
-		},
-		vec![],
-		None,
-		None,
-		Some(props),
-		Extensions {
-			relay_chain: "local_testnet".into(),
-			para_id: para_id.into(),
-		},
-	)
+    ChainSpec::from_genesis(
+        "Artemis Local Testnet",
+        "local_testnet",
+        ChainType::Local,
+        move || {
+            testnet_genesis(
+                vec![
+                    get_account_id_from_seed::<sr25519::Public>("Alice"),
+                    get_account_id_from_seed::<sr25519::Public>("Bob"),
+                    get_account_id_from_seed::<sr25519::Public>("Charlie"),
+                    get_account_id_from_seed::<sr25519::Public>("Dave"),
+                    get_account_id_from_seed::<sr25519::Public>("Eve"),
+                    get_account_id_from_seed::<sr25519::Public>("Ferdie"),
+                    get_account_id_from_seed::<sr25519::Public>("Relay"),
+                    get_account_id_from_seed::<sr25519::Public>("Alice//stash"),
+                    get_account_id_from_seed::<sr25519::Public>("Bob//stash"),
+                    get_account_id_from_seed::<sr25519::Public>("Charlie//stash"),
+                    get_account_id_from_seed::<sr25519::Public>("Dave//stash"),
+                    get_account_id_from_seed::<sr25519::Public>("Eve//stash"),
+                    get_account_id_from_seed::<sr25519::Public>("Ferdie//stash"),
+                ],
+                para_id,
+            )
+        },
+        vec![],
+        None,
+        None,
+        Some(props),
+        Extensions {
+            relay_chain: "local_testnet".into(),
+            para_id: para_id.into(),
+        },
+    )
 }
 
 /// Configure initial storage state for FRAME modules.
-fn testnet_genesis(
```

Comment:
ditto. See comment above. This formatting change is unnecessary.

Reply:
https://github.com/Snowfork/polkadot-ethereum/pull/442/commits/a31d23b3ae9b5d3f5a697ddb62b138c0c3225ad5 and https://github.com/Snowfork/polkadot-ethereum/pull/442/commits/8f9accfdf79d7b5bf931ae623a8eb55fbfc8af3c